exercises/protein-translation/canonical-data.json

```diff
@@ -0,0 +1,143 @@
+{
+  "exercise": "protein-translation",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Translate input RNA sequences into proteins",
+      "comments": [
+        "Returns the name of the protein if given RNA is valid, "
+      , "else throws an error. "
+      ],
+      "cases": [
+        {
+          "description": "Methionine RNA sequence is identified",
+          "property": "translate",
+          "strand": "AUG",
+          "expected": ["Methionine"]
+        },
+        {
+          "description": "Phenylalanine RNA sequence 1 is identified",
+          "property": "translate",
+          "strand": "UUU",
+          "expected": ["Phenylalanine"]
+        },
+        {
+          "description": "Phenylalanine RNA sequence 2 is identified",
```

Comment:
I am wondering if there is another way to discern between the different Phenylalanine types? They are currently named "Phenylalanine RNA sequence 1" and "Phenylalanine RNA sequence 2", but maybe there is a more descriptive name? If there isn't, just ignore this :)

Reply:
From the problem description, I couldn't see a particularly good way of differentiating the two from each other - maybe if someone knows biology they could chime in, otherwise this might be as good as we can get it.